src/source/PeerConnection/Retransimitter.c

```diff
@@ -59,24 +59,20 @@ STATUS resendPacketOnNack(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerCo
     CHK_STATUS(rtcpNackListGet(pRtcpPacket->payload, pRtcpPacket->payloadLength, &senderSsrc, &receiverSsrc, NULL, &filledLen));
 
     CHK_STATUS(doubleListGetHeadNode(pKvsPeerConnection->pTransceievers, &pCurNode));
-    while(pCurNode != NULL && (pReceiverTransceiver == NULL || pSenderTranceiver == NULL)) {
+    while(pCurNode != NULL && (pSenderTranceiver == NULL)) {
```

Comment:
no need for parenthesis around (pSenderTranseiver == NULL) as you've removed the ||